packages/ui-components/.babelrc.js

```diff
@@ -1,42 +0,0 @@
-/*
- * Tupaia
- * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
- */
-
-const isTest = process.env.NODE_ENV === 'test';
```

Comment:
It seems that this was not used... I deleted it to avoid confusion

Reply:
Oh, thanks!